src/blosc2/ndarray.py

```diff
@@ -745,9 +772,45 @@ def get_chunk(self, nchunk):
         """Shortcut to :meth:`SChunk.get_chunk <blosc2.schunk.SChunk.get_chunk>`. This can be accessed
         through the :attr:`schunk` attribute as well.
 
+        Parameters
+        ----------
+        nchunk : int
+        The index of the chunk to retrieve.
+
+        Returns
+        -------
+        chunk : array-like
+            The chunk data at the specified index. The type of this data depends on how the chunks are stored and
+            represented in the `SChunk` object.
```

Comment:
I'd remove the last sentence, as it does not bring too much interesting info.

Reply:
Done!